server/services/v1/api.go

```diff
@@ -212,8 +216,10 @@ func (s *apiService) Update(ctx context.Context, r *api.UpdateRequest) (*api.Upd
 
 	return &api.UpdateResponse{
 		Status:        resp.status,
-		Timestamp:     resp.timestamp.GetProtoTS(),
 		ModifiedCount: resp.modifiedCount,
+		Metadata: &api.ResponseMetadata{
```

Comment:
In internal/data.proto there is a timestamp struct we need to add created_at/updated_at in it otherwise we will not have created_at inside the FDB for the documents.

Reply:
fixed.